deploy/docs/Best_Practices.md

```diff
@@ -1095,3 +1098,55 @@ The reliability of the system can be calculated using the following formulas:
+
+Kubernetes offers a feature of assigning specific pod to node. Such kind of control is sometimes useful,
+whenever you want to ensure that pod will end up on specific node according your requirements like operating system
+or connected devices.
+
+### Binding pods to linux nodes
+
+Using this feature we can bind them to linux nodes.
+In order to do that `nodeSelector` has to be used. By default node selectors can be set for below pods:
+
+| component                   | key                                                                            | value |
+|-----------------------------|--------------------------------------------------------------------------------|-------|
+| `fluent-bit`                | `fluent-bit.nodeSelector.kubernetes.io/os`                                     | linux |
+| `fluentd`                   | `fluentd.events.statefulset.nodeSelector.kubernetes.io/os`                     | linux |
+| `fluentd`                   | `fluentd.logs.statefulset.nodeSelector.kubernetes.io/os`                       | linux |
+| `sumologic`                 | `sumologic.setup.job.nodeSelector.kubernetes.io/os`                            | linux |
+| `kube-prometheus-stack`     | `kube-prometheus-stack.prometheus-node-exporter.nodeSelector.kubernetes.io/os` | linux |
+| `kube-state-metrics`        | `kube-state-metrics.nodeSelector.kubernetes.io/os`                             | linux |
+| `prometheus`                | `prometheus.prometheusSpec.nodeSelector.kubernetes.io/os`                      | linux |
+
+Node selector can be changed via additional parameter to helm upgrade command, below example for fluent-bit node selector:
+
+```bash
+helm upgrade --install my-release sumologic/sumologic \
+  --set sumologic.accessId="<SUMO_ACCESS_ID>" \
+  --set sumologic.accessKey="<SUMO_ACCESS_KEY>" \
+  --set sumologic.clusterName="<MY_CLUSTER_NAME>" \
+  --set fluent-bit.nodeSelector."kubernetes\.io/os"=linux 
+```
+
+or by modyfing above settings in values.yaml
+
+```yaml
+fluent-bit:
+  nodeSelector:
+    kubernetes.io/os: linux
+```
+
+Node selectors for other pod types can be set in the same way.
+
+| component                   | key                                                          | value |
+|-----------------------------|--------------------------------------------------------------|-------|
+| `otelagent`                 | `otelagent.daemonset.nodeSelector.kubernetes.io/os`          | linux |
+| `otelcol`                   | `otelcol.deployment.nodeSelector.kubernetes.io/os`           | linux |
+| `otelgateway`               | `otelgateway.deployment.nodeSelector.kubernetes.io/os`       | linux |
+| `otellogs`                  | `otellogs.daemonset.nodeSelector.kubernetes.io/os`           | linux |
+| `metadata`                  | `metadata.metrics.statefulset.nodeSelector.kubernetes.io/os` | linux |
+| `metadata`                  | `metadata.logs.statefulset.nodeSelector.kubernetes.io/os`    | linux |
```

Comment:
Why are they two tables?

Reply:
The second table contains all other components that have nodeSelector label. I don't know if this table should be here or not. That's the question

Reply:
I would merge them into one table. I don't see a point in making two groups of pods